src/Altinn.Common.PEP/Altinn.Common.PEP/Constants/AltinnXacmlUrns.cs

```diff
@@ -39,5 +39,15 @@ public static class AltinnXacmlUrns
         /// Get the value task id
         /// </summary>
         public const string TaskId = "urn:altinn:task";
+
+        /// <summary>
+        /// Get the value resourceId
+        /// </summary>
+        public const string ResourceId = "urn:resourceregistry:resourceid";
```

Comment:
So far for changes made in access-management we have used the urn: urn:altinn:resourceregistry
as AttributeId for match for the resourceid. We should decide on a common value here

Reply:
ResourceId is also a bit generic property name. Should this be ResourceRegistryId?

Reply:
I changed to match your value in access-management